deploy/helm/edc-operator/crds/crds.yaml

```diff
@@ -32,21 +32,22 @@ spec:
                     ionos:
                       properties:
                         s3:
-                          description: Operators are expected to define fields for this type in order to work with S3 buckets.
+                          description: An S3 bucket definition, it can either be a reference to an explicit S3Bucket object, or it can be an inline defintion of a bucket. Read the [S3 resources concept documentation](https://docs.stackable.tech/home/nightly/concepts/s3) to learn more.
```

Comment:
Damn :D

Reply:
As this not critical at all, I would say keep this PR as is and incorporate the fix for the typo in a new bump PR in the future. I will open a PR for `operator-rs`.

Reply:
Fix-up PR: https://github.com/stackabletech/operator-rs/pull/710